public/js/main.js

```diff
@@ -92,13 +93,14 @@ if (connTarget) {
   });
 } else if (isFirefoxPanel()) {
   // The toolbox already provides the tab to debug.
-  module.exports = {
-    setThreadClient: firefox.setThreadClient,
-    setTabTarget: firefox.setTabTarget,
-    initPage: firefox.initPage,
-    getActions: () => actions,
-    renderApp: () => renderRoot(App)
-  };
+  function bootstrap({ threadClient, tabTarget }, callback) {
```

Comment:
hmm, i don’t see this on master.

Are you sure this is not an old PR?


On Thu, Sep 1, 2016 at 12:39 PM James Long <notifications@github.com> wrote:

> In public/js/main.js
> <https://github.com/devtools-html/debugger.html/pull/651#discussion_r77178235>
> :
>
> > @@ -92,13 +93,14 @@ if (connTarget) {
> >    });
> >  } else if (isFirefoxPanel()) {
> >    // The toolbox already provides the tab to debug.
> > -  module.exports = {
> > -    setThreadClient: firefox.setThreadClient,
> > -    setTabTarget: firefox.setTabTarget,
> > -    initPage: firefox.initPage,
> > -    getActions: () => actions,
> > -    renderApp: () => renderRoot(App)
> > -  };
> > +  function bootstrap({ threadClient, tabTarget }, callback) {
>
> I don't think we need the callback argument?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/devtools-html/debugger.html/pull/651/files/4811709861b1663416c5d938cb44446b8881c5a6#r77178235>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPiYpuZnbxStn1VTXuf4WhsrWapBhJVks5qltmngaJpZM4JyHNV>
> .
>